Source/WebKitLegacy/mac/DOM/DOMCSSRule.h

```diff
@@ -30,6 +30,7 @@
 @class NSString;
 
 enum {
+    DOM_UNKNOWN_RULE = 0,
```

Comment:
It's questionable that this is even needed, but avoiding breaking external users is worthwhile I suppose -- even for something deprecated in late 2018.